### DISCUSSION TITLE
Is this possible to use in CTGP-7

### DISCUSSION THREAD

**Discussion by shawny16k** [xzn/ntr-hr#11] [General]
The mod you've made is wonderful and all, the only small issue I have is I want to use this while playing CTGP7 as per their ntr viewer isn't really the best and I rather stream with this one, the developer of the CTGP probably hasn't know or doesn't seem to care to implement this awesome feature. My question is can I replace some files to get this to work?

**Comment by xzn** [comment]
CTGP-7 uses their own version of NTR, I don't see how it can be made to work. My version has been rewritten with many features (non-streaming stuff) removed. It will be a lot effort to integrate it even if the developers of CTGP-7 want to.

One thing I can try is to hide NTR-HR's presence from CTGP-7, probably by changing the address of where NTR-HR is loaded. Not sure if this is a good idea though..

**Reply by shawny16k** [reply]
CTGP detects any NTR if its not loaded by their own, not sure if there is a workaround to that.
